src/main/java/domain/CardDeck.java

```diff
@@ -0,0 +1,63 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CardDeck {
+    public static final int MAX_SCORE = 21;
+    private static final int ACE_MAX = 11;
+    private static final int ACE_MIN = 1;
+
+    private final List<TrumpCard> cards;
+
+    public CardDeck() {
+        this.cards = new ArrayList<>();
+    }
+
+    public void addTrumpCard(TrumpCard card) {
+        cards.add(card);
+    }
+
+    public List<TrumpCard> getFirstCard() {
+        if (cards.isEmpty()) {
+            throw new IllegalArgumentException("카드가 비었습니다.");
+        }
+        return List.of(cards.getFirst());
+    }
+
+    public List<TrumpCard> getAllCard() {
+        if (cards.isEmpty()) {
+            throw new IllegalArgumentException("카드가 비었습니다.");
+        }
+        return new ArrayList<>(cards);
+    }
+
+    public boolean hasAce() {
+        return cards.stream()
+            .anyMatch(card -> card.getCardNumber() == CardNumber.ACE);
```

Comment:
```suggestion
            .anyMatch(card -> card.isAce());
```

card의 값을 꺼내서 비교하기보단 객체에 메세지를 보내서 캡슐화를 지켜보는건 어떨까요?

Reply:
TrumpCard내부에 isAce메서드를 생성하여 해결해 보았습니다!